### PR TITLE
UX Changes For Beta

### LINKED ISSUE #566: Add error message for lack of mobile support

### PR DESCRIPTION
**Pull request by aryanshah701**
# Description

A bunch of beta related changes + bug fixes:
- Disable app on views < 1100px window inner width
- Landing page and log updates
- Fix .env issue in api-v2
- Minor updates to email confirmation text
- Improve error handling for email confirmation
- Display warning to users with un-comfirmed emails in Account Overview and allow them to have the email be resent
- Prevent leading or trailing whitespaces in string inputs in forms
- Display exceptions for ranges in sidebar
- Display section warnings in sidebar

Closes #566 #550 #602 #528 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually. For sidebar stuff, updated BSCS to ensure stuff works.

# Checklist:

- [x] I have run the production builds in docker for the frontend/backend and ensure things run fine. Check README of repo on how to run if not sure.
- [x] I have performed a self-review of my own code
- [x] I have commented my code where needed
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jia614 — APPROVED** [review]
🚢 it

File: packages/frontend-v2/utils/validation/noLeadOrTrailWhitespacePattern.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ValidationRule } from "react-hook-form";
```
Comment (BrandonLim8890) [inline packages/frontend-v2/utils/validation/noLeadOrTrailWhitespacePattern.ts:1]: shift this to common because it could be used by BE as well?

**Review by BrandonLim8890 — COMMENTED** [review]
LGTM

File: packages/frontend-v2/utils/validation/noLeadOrTrailWhitespacePattern.ts (hunk 1 above)
Comment (aryanshah701) [reply packages/frontend-v2/utils/validation/noLeadOrTrailWhitespacePattern.ts:1]: truuuue, but for now this is coupled to react-hook-forms only. We can standardize and move to back end later, might just move the regex itself.

**Pull request merged**